vignettes/sir_incidence_data.txt

```diff
@@ -0,0 +1 @@
+3,2,2,1,1,4,2,5,6,6,8,4,5,7,8,13,14,13,16,14,16,25,17,17,21,18,16,15,16,19,12,11,19,11,22,13,20,15,19,17,22,23,14,8,13,10,20,15,9,14,6,8,5,9,7,1,6,7,2,7,8,5,4,7,2,8,6,3,1,2,5,2,0,3,2,1,1,1,2,3,1,3,0,2,1,2,0,0,2,0,4,1,1,2,0,2,1,0,1,0
```

Comment:
As discussed on teams:

* can this please be saved as a .csv with both columns
* can the script to generate it be put into the `scripts` directory; write as a standalone .R file that runs assuming that it is run from the package root. You can use `pkgload::load_all()` to load internal functions if needed